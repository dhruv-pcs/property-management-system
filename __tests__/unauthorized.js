import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import Unauthorized from 'src/pages/unauthorized'

describe('Unauthorized Component', () => {
  it('should render unauthorized component', async () => {
    const { container } = render(<Unauthorized />)
    expect(container).toMatchSnapshot()
  })
})
