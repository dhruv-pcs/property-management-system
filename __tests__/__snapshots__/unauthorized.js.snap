// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized Component should render unauthorized component 1`] = `
<div>
  <div
    class="d-flex flex-column align-items-center justify-content-center "
    style="height: 100vh; background-color: rgb(255, 255, 255);"
  >
    <h1>
      Unauthorized
    </h1>
    <p>
      You are not authorized to access this page.
    </p>
    <a
      aria-label="Dashboard"
      class="btn btn-light"
      href="/"
      style="background-color: rgb(76, 206, 172);"
    >
      Go to Dashboard
    </a>
  </div>
</div>
`;
